static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -19,6 +20,27 @@
                     maxlength="1000">{{ realm_description }}</textarea>
             </div>
             <div class="input-group">
+                <label for="realm_notifications_stream" id="realm_notifications_stream_label" class="inline-block"
+                    title="{{t 'The stream to which new stream notifications go to.' }}">
+                    {{t "Notifications Stream: " }}
+                    <span id="realm_notifications_stream_name"></span>
+                </label>
+                {{#if is_admin }}
+                <span class="dropup actual-dropdown-menu" id="id_realm_notifications_stream"
+                    name="realm_notifications_stream" aria-labelledby="realm_notifications_stream_label"
+                    data-selected-stream-id={{realm_notifications_stream_id}}>
+                    <a class="dropdown-toggle" data-toggle="dropdown">{{t "[Change]" }}</a>
+                    <ul class="dropdown-menu" role="menu">
+                        <li class="dropdown-search" role="presentation">
+                            <input type="text" role="menuitem" placeholder="{{t 'Filter streams' }}" autofocus/>
+                        </li>
+                        <span class="dropdown-list-body"></span>
+                    </ul>
+                </span>
+                <a class="notifications-stream-disable">{{t "[Disable]" }}</a>
+                {{/if}}
+            </div>
+            <div class="input-group">
```

Comment:
I think we should probably move this setting to live below the "Save changes" button and instead just update the server as soon as user selects something.  It's pretty confusing as-is; I'd never remember to click the "Save changes" button.

(As a general note, the plan is to get rid of that button)